Topic2/src/main/java/Blogcase/Blog.java

```diff
@@ -0,0 +1,49 @@
+package Blogcase;
+
+import java.util.LinkedList;
+import java.util.List;
+
+import Blogcase.Entry;
+
+public class Blog {
+
+	private Entry entry;
+
+	private List<Entry> entryList;
+
+	public void newList() {
+		entryList = new LinkedList<Entry>();
+	}
+
+	public boolean newPost(Entry entry) {
+
+		entryList.add(entry);
+		return true;
+	}
+
+	public boolean deletePost(int id) { // we are suposing that the entry id is
```

Comment:
Don't assume order in collections, you can remove an object sending the object to delete.
https://docs.oracle.com/javase/8/docs/api/java/util/ArrayList.html#remove-java.lang.Object-